src/commands/chatters/winter.ts

```diff
@@ -0,0 +1,22 @@
+import { Discord, SimpleCommand, SimpleCommandMessage, SimpleCommandOption, SimpleCommandOptionType } from 'discordx'
+
+@Discord()
+class Winter {
+  @SimpleCommand({ name: 'winter', description: 'Winter', prefix: '!', argSplitter: '\n' })
```

Comment:
No need to include a prefix, the bot on init defaults our prefix to `>` in the rare chance we collide with brexbot